src/main/java/com/smalaca/messagesender/exceptions/NoMessageException.java

```diff
@@ -0,0 +1,15 @@
+package com.smalaca.messagesender.exceptions;
+
+public class NoMessageException extends RuntimeException {
+    private final String messageId;
+
+    public NoMessageException(String messageId) {
+        super("Message " + messageId + " doesn't exist");
+        this.messageId = messageId;
+    }
+
+    public String getMessageId() {
```

Comment:
jeżeli nigdzie nie wykorzystujecie tej metody to ją usuńcie. To implikuje również usunięcie atrybutu 😄